client/my-sites/patterns/components/readymade-templates/index.tsx

```diff
@@ -61,12 +65,16 @@ export const ReadymadeTemplates = () => {
 		return;
 	}
 
+	const props = forwardRef ? { forwardRef } : {};
+
 	return (
 		<PatternsSection
+			{ ...props }
 			bodyFullWidth
 			description={ translate(
-				'Explore a collection of beautiful site layouts made with our patterns.'
+				'Explore our beautiful site layouts made with patterns that you can fully customize to fit your needs.'
```

Comment:
This text has been updated here https://github.com/Automattic/wp-calypso/pull/92790

Reply:
Yup, I was looking at your PR actually, haha. I'll remove these changes in favor of yours.